go/common/errutil/errors_util.go

```diff
@@ -1,8 +1,12 @@
 package errutil
 
-import "errors"
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum"
+)
 
 var (
-	ErrNotFound = errors.New("not found")
+	ErrNotFound = ethereum.NotFound
```

Comment:
Can you put a comment above this saying why it's important that it's an alias just on the off-chance someone does try and change this.